### PR TITLE
Add content from Docker Tools wiki 

### PR DESCRIPTION
**Pull request by ghogen**

### CODE REVIEW

**Comment by ghogen** [comment]
@gregvanl @philliphoff

File: docs/containers/ssh.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+The simplest way is to use VS Code's [Remote - SSH](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh) extension, from the [Remote Development](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack) extension pack.
+
+1. Run command `Remote-SSH: Add new SSH host...` and follow the prompts to set up a connection to the target host.
+
+1. Run command `Remote-SSH: Connect to host...` and connect to the host.
+
+1. A new VS Code window opens, remoted to the target machine. If using password authentication, the password will be prompted here. It is recommended to set up [SSH key authentication](https://www.ssh.com/ssh/public-key-authentication), for ease of use. In the Extensions tab, install the Docker extension (on the remote host) (a reload may be required after this step):
+
+![image](https://user-images.githubusercontent.com/36966225/66958480-3f280b80-f036-11e9-8d75-b4e55eb3913f.png)
```
Comment (gregvanl) [inline docs/containers/ssh.md]: Add local version of the image
Also need descriptive alt-text for image with users with screen readers.

File: docs/containers/debug-common.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+The Docker extension provides a `docker` debug configuration provider that manages how VS Code will launch an application and/or attach a debugger to the application in a running Docker container. This provider is configured via entries within `launch.json`, with configuration being specific to each application platform supported by the provider.
+
+## Platform Support
+
+The Docker extension currently supports debugging .NET Core and Node.js applications within Docker containers. 
+
+### .NET Core
+
+More information about debugging .NET Core applications within Docker containers can be found [here](/docs/containers/debug-netcore.md).
```
Comment (gregvanl) [inline docs/containers/debug-common.md]: Avoid [here] for users with screen readers.
"can be found in [Debug .NET Core within Docker containers].

File: docs/containers/choosing-dev-environment.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    ...
+    "runArgs": [ "-v", "/var/run/docker.sock:/var/run/docker.sock"]
+    ...
+```
+
+### Windows Subsystem for Linux
+
+Windows Subsystem for Linux represents a great choice for container-based service development on Windows. [Windows Subsystem for Linux version 2 (WSL 2)](https://docs.microsoft.com/windows/wsl/wsl2-index) is strongly recommended. Docker Desktop for Windows has been updated to work with WSL 2 and has a graphical setting to enable Docker CLI inside WSL 2 distribution(s):
+
+![Enable Docker inside WSL 2 distribution](images/devenv/devenv-enable-docker-wsl2.png)
```
Comment (gregvanl) [inline docs/containers/choosing-dev-environment.md]: broken image link

File: docs/containers/app-service.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+- Create a container image for your application.
+- Push the image to a container registry.
+- Deploy the image to Azure App Service.
+
+## Prerequisites
+
+- An Azure subscription.
+- [Docker](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker) and [Azure App Service](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureappservice) extensions must be installed.
+- A [**web** application](https://docs.microsoft.com/azure/app-service/containers/tutorial-custom-docker-image) that produces a docker image. You could also follow [Create a sample ASP .NET Core application](ASP-Net-Core) to create such application.
```
Comment (gregvanl) [inline docs/containers/app-service.md]: missing link target

File: docs/containers/debug-common.md (hunk 2 above)
Comment (gregvanl) [reply docs/containers/debug-common.md]: I think there are a couple more instances of [here] in this topic.

Comment (ghogen) [reply docs/containers/debug-common.md]: Thanks, got those now.

**Pull request merged**